cvmfs/platform_linux.h

```diff
@@ -92,10 +89,8 @@ inline bool platform_umount(const char* mountpoint, const bool lazy) {
       return false;
     }
     FILE *fmntnew = setmntent(mntnew.c_str(), "w+");
-    if (!fmntnew &&
-        (chmod(mntnew.c_str(), mtab_info.st_mode) != 0) &&
-        (chown(mntnew.c_str(), mtab_info.st_uid, mtab_info.st_gid) != 0))
-    {
+    if (!fmntnew && (chmod(mntnew.c_str(), mtab_info.st_mode) != 0) &&
```

Comment:
I wondered for a while, are these reformattings automatically done by the editor?  I wondered because they add a number of "things" to the pull requests besides the actual changes.